spec/features/password_length_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'simple_send_keys'
+
+RSpec.feature "new user password", type: :feature, js: true do
+  scenario "user cannot input password more than 10 characters" do
+    visit('/')
+    click_link('Sign up')
+    fill_in('user_email', :with => 'tester@test.com')
+    fill_in('user_password', :with => 'reallyreallylongpassword')
+    fill_in('user_password_confirmation', :with => 'reallyreallylongpassword')
+    click_button 'Sign up'
+    expect(page.current_path).to eq '/users'
+    expect(page).to have_content("Password is too long (maximum is 10 characters)")
+  end
+
+  scenario "user cannot input password less than 6 characters" do
+    visit('/')
+    click_link('Sign up')
+    fill_in('user_email', :with => 'tester@test.com')
+    fill_in('user_password', :with => 'short')
+    fill_in('user_password_confirmation', :with => 'short')
+    click_button 'Sign up'
+    expect(page.current_path).to eq '/users'
+    expect(page).to have_content("Password is too short (minimum is 6 characters)")
+  end
+end
+
+# RSpec.feature "Timeline", type: :feature, js: true do
```

Comment:
Can we delete this bit before merging?

Reply:
Yes, you mean the rogue comment (line 28)?

Reply:
yes, the one that is commented out